src/sentry/static/sentry/app/views/settings/project/projectUserFeedback.jsx

```diff
@@ -0,0 +1,186 @@
+import Panel from '../components/panel';
+import PanelBody from '../components/panelBody';
+import PanelHeader from '../components/panelHeader';
+import SettingsPageHeader from '../components/settingsPageHeader';
+import TextBlock from '../components/text/textBlock';
+import formGroups from '../../../data/forms/userFeedback';
+
+const CodeBlock = styled.pre`
+  word-break: break-all;
+  white-space: pre-wrap;
+`;
+
+const TextBlockNoMargin = styled(TextBlock)`
+  margin-bottom: 0;
+`;
+
+class ProjectUserReportSettings extends AsyncView {
+  static propTypes = {
+    setProjectNavSection: PropTypes.func,
+  };
+
+  componentWillMount() {
+    super.componentWillMount();
+    this.props.setProjectNavSection('settings');
+  }
+
+  componentDidMount() {
+    window.sentryEmbedCallback = function(embed) {
+      // Mock the embed's submit xhr to always be successful
+      // NOTE: this will not have errors if the form is empty
+      embed.submit = function(body) {
+        this._submitInProgress = true;
+        setTimeout(
+          function() {
+            this._submitInProgress = false;
+            this.onSuccess();
+          }.bind(this),
+          500
+        );
+      };
+    };
+  }
+
+  componentWillUnmount() {
+    window.sentryEmbedCallback = null;
+  }
+
+  getEndpoints() {
+    let {orgId, projectId} = this.props.params;
+    return [
+      ['keyList', `/projects/${orgId}/${projectId}/keys/`],
+      ['project', `/projects/${orgId}/${projectId}/`],
+    ];
+  }
+
+  getInstructions() {
+    let dsn = this.state.keyList.length
+      ? this.state.keyList[0].dsn.public
+      : 'https://public@sentry.example.com/1';
+
+    return (
+      '<!-- Sentry JS SDK 2.1.+ required -->\n' +
+      '<script src="https://cdn.ravenjs.com/2.1.0/raven.min.js"></script>\n\n' +
+      '{% if request.sentry.id %}\n' +
+      '  <script>\n' +
+      '  Raven.showReportDialog({\n' +
+      '    // grab the eventId generated by the Sentry SDK\n' +
+      "    eventId: '{{ request.sentry.id }}',\n\n" +
+      '    // use the public DSN (dont include your secret!)\n' +
+      "    dsn: '" +
+      dsn +
+      "'\n" +
+      '  });\n' +
+      '  </script>\n' +
+      '{% endif %}\n'
+    );
+  }
+
+  getBrowserJSInstructions() {
+    let dsn = this.state.keyList.length
+      ? this.state.keyList[0].dsn.public
+      : 'https://public@sentry.example.com/1';
+
+    return (
+      '<!-- Sentry JS SDK 2.1.+ required -->\n' +
+      '<script src="https://cdn.ravenjs.com/2.1.0/raven.min.js"></script>\n\n' +
+      '<script>\n' +
+      '// configure the SDK as you normally would\n' +
+      "Raven.config('" +
+      dsn +
+      "').install();\n\n" +
+      '/**\n' +
+      ' * Report a routing error to Sentry and show a feedback dialog to\n' +
+      ' * the user.\n' +
+      ' * \n' +
+      ' * > try {\n' +
+      ' * >   renderRoute()\n' +
+      ' * > } catch (err) {\n' +
+      ' * >   handleRouteError(err);\n' +
```

Comment:
Again, not necessary, but it'd be great to have this reflect a React error boundary example.